src/ProjectNeon/Assets/Features/Effects/Attack.cs

```diff
@@ -1,18 +1,18 @@
 ﻿
-
-
 using UnityEngine;
 
 public sealed class Attack  : Effect
 {
 
-    public Member Attacker { get; private set; }
-    public Target Target { get; private set; }
-    public int Damage { get; }
+    public Member Attacker { get; set; }
+    public Target Target { get; set; }
+    public int Damage { get; set; }
```

Comment:
Damage amount can't be known until after the attack has resolved, or at least after Targets have been finalized.